### DISCUSSION TITLE
How npx create llama gives a response? | next.js

### DISCUSSION THREAD

**Discussion by 9mean2** [run-llama/LlamaIndexTS#867] [General]
I created create-llama with Next.js.
But I don't think websockets or sse are used here.
However, I am curious about how to receive the response in real time.

It's my first time using next.js

**Comment by marcusschiesser** [comment]
It's using Vercel/AI protocol for streaming, here are some details: https://vercel.com/blog/an-introduction-to-streaming-on-the-web
